tools/container/toolkit/toolkit.go

```diff
@@ -404,7 +417,65 @@ func Install(cli *cli.Context, opts *options) error {
 		log.Errorf("Ignoring error: %v", fmt.Errorf("error installing NVIDIA Container CDI Hook CLI: %v", err))
 	}
 
-	err = installToolkitConfig(cli, toolkitConfigPath, nvidiaContainerCliExecutable, nvidiaCTKPath, nvidiaContainerRuntimeHookPath, opts)
+	var runtimeBinaryPaths []string
+
+	if r, ok := os.LookupEnv("RUNTIME"); ok {
```

Comment:
See #704 where we would be able to pass this to the install function instead.

It may be cleaner regardless to add a runtime command line argument instead.